symbols/overlay29.yml

```diff
@@ -1360,6 +1368,17 @@ overlay29:
         Reveals traps within the monster's viewing range.
         
         r0: entity pointer
+    - name: ShouldMonsterRunAI
+      address:
+        NA: 0x22EF9EC
+      description: |-
+        Checks a monster's monster_behavior to see whether or not the monster should use AI. Only called on monsters with
+        a monster_behavior greater than or equal to 6. Returns false for BEHAVIOR_FIXED_ENEMY, 
+        BEHAVIOR_WANDERING_ENEMY_0x8, BEHAVIOR_SECRET_BAZAAR_KIRLIA, BEHAVIOR_SECRET_BAZAAR_MIME_JR,
+        BEHAVIOR_SECRET_BAZAAR_SWALOT, BEHAVIOR_SECRET_BAZAAR_LICKILICKY, and BEHAVIOR_SECRET_BAZAAR_SHEDINJA.
+        
+        r0: monster entity pointer
+        return: bool
```

Comment:
It's not related to running away it's called in RunMonsterAi, should I change it to ShouldRunMonsterAi for clarity? If the function returns 0, the RunMonsterAi function will end without doing anything. I assume it's done so that the bazaar monsters will not move, but for the others it returns false for I'm not sure.

Reply:
Oh, I see now. The current name is a bit confusing. I think `ShouldRunMonsterAi` would be better.